src/services/exile_service.py

```diff
@@ -108,15 +108,13 @@ async def unexile_user(
     # update exile record
     db_user = users_database.get_user(user.id)
     if db_user is None:
+        error_message = "User does not exist in database, does not have any exiles, no action will be taken"
```

Comment:
i think we can just shorten this string to `User does not have any exiles, no action will be taken`. if the user doesn't exist in the db that means they don't have an exile to begin with